Backup Restore

zanarkandlee opened the pull request.

zanarkandlee committed:
Backup Restore

wilsontanwm commented on traitquestdev/css/header.css:
```diff
@@ -0,0 +1,25 @@
+header {
```
i think this file is not used anymore? can remove?

wilsontanwm commented on traitquestdev/css/ohsemquestheader.css:
```diff
@@ -0,0 +1,13 @@
+header {
```
i think this file is not used anymore? can remove?

wilsontanwm commented on traitquestdev/css/ohsemquestregistration.css:
```diff
@@ -0,0 +1,46 @@
+
```
i think this file is not used anymore? can remove?

wilsontanwm commented on traitquestdev/css/signup.css:
```diff
@@ -0,0 +1,89 @@
+body {
```
i think this file is not used anymore? can remove?

zanarkandlee committed:
Backup_Restore_update

-deleted files as per requested in github comments
-edited padding in index page
-updated style.css and added setheight.js for admin pages

zanarkandlee committed:
Backup_Restore_update2

zanarkandlee committed:
Backup_Restore_update3

test description
